### ISSUE TITLE
Detector pixel coordinate specification isn't necessarily consistent with odd/even array sizes

### ISSUE THREAD

**Issue by mperrin** [mperrin/webbpsf#200]
As discussed with @shanosborne 

Consider:
```
fgs = FGS()
fgs.detector_position= (100,400)
psf = fgs.calc_psf(fov_pixels=100)
```
The resulting PSF has even dimensions so it's exact center is implicitly on a half pixel boundary. The detector position coordinates written into the FITS header should probably be modified to reflect that. 

Could also consider giving the mapping to the detector pixel coordinates of the first pixel (data[0,0]) in the array?  Need to be more thoughtful about the precise consistency of this with the SIAF values.

**Comment by mperrin** [comment]
@shanosborne is fixing this

**Comment by mperrin** [comment]
Fixed by #205
